src/PrestaShopBundle/Entity/Repository/ShipmentRepository.php

```diff
@@ -43,6 +44,11 @@ public function findByOrderId(int $orderId)
         return $this->findBy(['orderId' => $orderId]);
     }
 
+    public function findByShipmentId(int $shipmentId): ?Shipment
```

Comment:
What is the naming rule when you return an instance of an object from its primary key? Isn't it

```suggestion
    public function findById(int $shipmentId): ?Shipment
```